src/components/OldApp/actions.js

```diff
@@ -0,0 +1,22 @@
+import {SET_USERS, SET_SLACK_USER_FETCH_ERROR} from './actionTypes';
+
+import {loadUsers as reallyLoadUsers} from './../../utils/slack';
+
+export function setUsers(users) {
+	return {
+		type: SET_USERS,
+		users
+	};
+}
+
+export function setSlackUserFetchError(slackUserFetchError) {
+	return {
+		type: SET_SLACK_USER_FETCH_ERROR,
+		slackUserFetchError
+	};
+}
+
+export const loadUsers = () => (dispatch, getState) =>
+	reallyLoadUsers()
```

Comment:
Wasn't this renamed to fetchUsers as part of another piece of work yesterday?

Reply:
yes, not sure what happened - this is fixed in the Routing branch